src/sonic-device-data/tests/hwsku_json_checker

```diff
@@ -7,7 +7,7 @@ import sys
 
 # Global variable
 PORT_ATTRIBUTES = ["default_brkout_mode"]
-OPTIONAL_PORT_ATTRIBUTES = ["fec", "autoneg"]
+OPTIONAL_PORT_ATTRIBUTES = ["fec", "autoneg","port_type"]
```

Comment:
Add space after comma.

Reply:
Joe - given that platform.json has been expanded to include "breakout_modes", "index", etc... would it make more sense to add "port_type" there instead of hwsku.json

Reply:
It depends on whether this is something that is fixed at the platform level and will apply to all HwSKUs configured on that platform, or whether it will vary by HwSKU.